src/Http/Http.Extensions/test/ParameterBindingMethodCacheTests.cs

```diff
@@ -380,6 +380,23 @@ public void FindTryParseMethod_ThrowsIfInvalidTryParseOnType(Type type)
         Assert.Contains($"bool TryParse(string, out {TypeNameHelper.GetTypeDisplayName(type, fullName: false)})", ex.Message);
     }
 
+    [Theory]
+    [InlineData(typeof(InvalidVoidReturnTryParseStruct))]
+    [InlineData(typeof(InvalidVoidReturnTryParseClass))]
+    [InlineData(typeof(InvalidWrongTypeTryParseStruct))]
+    [InlineData(typeof(InvalidWrongTypeTryParseClass))]
+    [InlineData(typeof(InvalidTryParseNullableStruct))]
+    [InlineData(typeof(InvalidTooFewArgsTryParseStruct))]
+    [InlineData(typeof(InvalidTooFewArgsTryParseClass))]
+    [InlineData(typeof(InvalidNonStaticTryParseStruct))]
+    [InlineData(typeof(InvalidNonStaticTryParseClass))]
+    [InlineData(typeof(TryParseWrongTypeInheritClass))]
+    [InlineData(typeof(TryParseWrongTypeFromInterface))]
```

Comment:
I'd turn these into `MemberData` so you could share the inputs between test and `FindTryParseMethod_ThrowsIfInvalidTryParseOnType`. Same suggestion for the other one.